test/VaultOUSD.js

```diff
@@ -0,0 +1,80 @@
+const { expect } = require("chai");
+const exp = require("constants");
+const { ethers } = require("hardhat");
+const {
+    BigNumber,
+    FixedFormat,
+    FixedNumber,
+    formatFixed,
+    parseFixed
+} = require("@ethersproject/bignumber");
+var helper = require('./MainnetHelper');
+const MainnetHelper = require("./MainnetHelper");
+
+describe("VaultOUSD test suit", function () {
+    let tokenVault;
+    let tokenOUSD;
+    let owner;
+    let addr1;
+    let addr2;
+    let addrs;
+    let sharesOwnerAddress;
+
+    let addr1Deposit = 10000
+    let addr2Deposit = 20000
+    let interestIntoVault = 10000
```

Comment:
OUSD decimal is 18 - are these numbers really small (like: 10,000 / 10^18) or these are 10K OUSD?
I would add a comment clarifying that (and if needed add the conversion to the correct decimal).
Otherwise we might forget what we meant next time we visit this test

Reply:
I am talking about 10k OUSD. Is the expectation for 10k OUSD to be returned and transfer via (10k*10^18)?

Reply:
OUSD is using 10^18 decimal
so the `transfer` function expects 10K*10^18 - if you keep it 10,000 just add a comment. It might be better to always use X*10^18 so we won't need to convert back and forth